src/util/Constants.js

```diff
@@ -7,13 +7,22 @@ export default {
     },
     KEYS: {
       GET_USER_LOGGED: 'GET_USER_LOGGED'
-    }
+    },
+    STALE_TIME: 1000 * 60 * 5
   },
   REQUEST: {
     GET: 'get',
     POST: 'get',
     PUT: 'put',
     PATCH: 'patch',
     DELETE: 'delete',
+  },
+  SESSION: {
```

Comment:
dont you think we could refactor this constant file and put it in place? like the stale_time was before? this way each variable would be on its context

Reply:
and again, we could create a task to refactor for all

Reply:
I think that moving constants to each component is the best approach as well.
I am going to create a task for it as well 😎